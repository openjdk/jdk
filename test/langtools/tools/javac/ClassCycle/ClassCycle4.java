/*
 * @test /nodynamiccopyright/
 * @bug 8320220
 * @summary Fix infinite recursion in cyclic inheritance situation
 * @compile/fail/ref=ClassCycle4.out -XDrawDiagnostics ClassCycle4.java
 */

interface ClassCycle4 extends I1, I2 {}
interface I1 extends ClassCycle4 {}
interface I2 extends ClassCycle4 {}
