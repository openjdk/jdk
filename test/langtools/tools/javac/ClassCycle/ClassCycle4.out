ClassCycle4.java:8:1: compiler.err.cyclic.inheritance: ClassCycle4
1 error
